src/routes/Accounts/AccountTabs/Overview/OverviewBalance.tsx

```diff
@@ -67,8 +67,8 @@ const OverviewBalance: FC<OverviewBalanceProps> = ({ account }) => {
                     state: { accountId: account.id },
                   })
                 }
-                isDisabled={!synced}
-                disabled={!synced}
+                isDisabled={account.viewOnly || !synced}
+                disabled={account.viewOnly || !synced}
```

Comment:
thought: Strange that both of these props exist 🤔

Reply:
Yea I have no idea why both are there, but I just added it here and moved on lol